terminologies/technique/maximumProbabilityMapping.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+ "@id":"https://openminds.ebrains.eu/controlledTerms/technique/maximumProbabilityMapping",

```

Comment:
I think this one we should discuss a bit more before we integrate it.

@tgbugs: what do you think?